qiskit_sphinx_theme/footer.html

```diff
@@ -69,3 +45,10 @@
 
 </footer>
 
+<script>
+  function clicked(ctaType) {
+    document.getElementById('was-helpful-thank-you').style.visibility = 'visible';
+    window.trackCta(`Helpful - ${ctaType}`);
+  }
+  </script>
```

Comment:
Nitpicking. Remove spaces

```suggestion
</script>
```